tools/releaseBuild/Images/microsoft_powershell_windowsservercore/PowerShellPackage.ps1

```diff
@@ -0,0 +1,104 @@
+[cmdletbinding()]
+# PowerShell Script to clone, build and package PowerShell from specified fork and branch
+param (
+    [string] $fork = 'powershell',
+    [string] $branch = 'master',
+    [string] $location = "$pwd\powershell",
+    [string] $destination = "$env:WORKSPACE",
+    [ValidateSet("win7-x64", "win81-x64", "win10-x64", "win7-x86")]    
+    [string]$Runtime = 'win10-x64',
+    [switch] $Wait,
+    [ValidatePattern("^v\d+\.\d+\.\d+(-\w+\.\d+)?$")]
+    [ValidateNotNullOrEmpty()]
+    [string]$ReleaseTag
+)
+
+$releaseTagParam = @{}
+if($ReleaseTag)
+{
+    $releaseTagParam = @{ 'ReleaseTag' = $ReleaseTag }
+}
+
+if(-not $env:homedrive)
+{
+    Write-Verbose "fixng empty home paths..." -Verbose
+    $profileParts = $env:userprofile -split ':'
+    $env:homedrive = $profileParts[0]+':'
+    $env:homepath = $profileParts[1]
+}
+
+if(! (Test-Path $destination))
```

Comment:
If you are using -Force for New-Item then you do not need to Test-Path.

Reply:
Removed the force...